clang/lib/StaticAnalyzer/Checkers/StreamChecker.cpp

```diff
@@ -1034,16 +1034,16 @@ void StreamChecker::preWrite(const FnDescription *Desc, const CallEvent &Call,
   C.addTransition(State);
 }
 
-static std::optional<QualType> getPointeeType(const MemRegion *R) {
+static QualType getPointeeType(const MemRegion *R) {
   if (!R)
-    return std::nullopt;
+    return {};
```

Comment:
It's a bit strange that here you return `{}`, while at the end of the function `QualType{}` is explicitly specified.